scoring_data.py

```diff
@@ -473,16 +473,17 @@ def _pretty_streak(streak_json_str):
     ),
     Category(
         "individual",
-        "Lowest XL Win",
-        "Vehumet values ruthless efficiency, and recognises the players who win at the lowest XL. Waiting around for an ancestor to return from memory is inefficient, as is dying repeatedly, so Felid games and games where Hepliaklqana is worshipped do not count in this category. For the purposes of this category, players who have not won and players who have won only at XL 27 are both ranked last.", "low_xl_win",
+        "Most Pacific Win",
+        "Elyvilon wishes for peace among the dungeon denizens and ranks the players on their wins with the fewest amount of kills. Monsters killing each other intentionally or otherwise is not considered pacific.",
```

Comment:
```suggestion
        "Elyvilon wishes for peace among the dungeon denizens and ranks the players on their wins with the fewest amount of kills. Monsters killing each other intentionally or otherwise is not considered pacific and is thus counted towards the kills as well.",
```